api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxNewCohortQueries.java

```diff
@@ -285,13 +291,13 @@ public CohortDefinition getTxNewUnionNumerator(CohortDefinition AgeCohort) {
 		queryParameters.put("startDrugsConcept", hivMetadata.getstartDrugsConcept().getConceptId());
 		queryParameters.put("historicalDrugsConcept", hivMetadata.gethistoricalDrugStartDateConcept().getConceptId());
 		
-		if (AgeCohort != null && AgeCohort instanceof AgeCohortDefinition) {
-			queryParameters.put("minAge", ((AgeCohortDefinition) AgeCohort).getMinAge());
-			queryParameters.put("maxAge", ((AgeCohortDefinition) AgeCohort).getMaxAge());
+		if (ageCohort instanceof AgeCohortDefinition) {
```

Comment:
@ynurmahomed why remove the null check for the ageCohort? This could be the cause of null exceptions down the line

Reply:
Its because `instanceof` will return `false` for `null` references, the null check is not necessary. Also IntelliJ was complaining :smile: